File: Code-Chef/Ceil-and-A-B-Problem/ceil_and_a-b_problem.c
/*  Problem Statement: https://www.codechef.com/problems/CIELAB
    Author: striker
*/

#include<stdio.h>
#include<stdlib.h>
#include<inttypes.h>
#include<assert.h>

int main(void) {
    uint32_t a,b;
    scanf("%"SCNu32"%"SCNu32, &a,&b);
    assert((a > 0 && a < 10001) && (b > 0 && b < 10001));
    if((a - b) % 10 == 9) {
        printf("%"PRIu32"\n", (a - b) - 1);
    } else {
        printf("%"PRIu32"\n", (a - b) + 1);
    }
    return 0;
}

File: Code-Chef/Ceil-and-A-B-Problem/ceil_and_a-b_problem.py
'''
Problem Statement: https://www.codechef.com/problems/CIELAB
Author: striker
'''

def main():
    a,b = tuple(map(int,input().split()))
    print((a - b) - 1) if (a - b) % 10 == 9 else print((a - b) + 1)

if __name__ == "__main__":
    main()